index.js

```diff
@@ -12,15 +12,15 @@ rangePlugin.install = function(Vue, options) {
         step = step || 1;
 
         if(typeof start !== 'number') {
-          console.warn(`Vue-for-range: provided argument for start - ${start} - is not a number.`)
+          console.warn('Vue-for-range: provided argument for start - ${start} - is not a number.')
```

Comment:
I've used [Temple literal syntax here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), so to make it work the same, please use string concatenation.

('Vue-for-range: provided argument for start -' + start + '- is not a number.')

Reply:
Didn't noticed that, my bad 🙂